src/Core/Framework/DataAbstractionLayer/Write/Command/ChangeSet.php

```diff
@@ -74,6 +74,13 @@ public function hasChanged(string $property): bool
         return \array_key_exists($property, $this->after) || $this->isDelete;
     }
 
+    public function merge(ChangeSet $changeSet): void
```

Comment:
Hi @jasperP98,

thanks for the bug fix, changes are looking good to me, but could you please provide some tests for it?

Integration tests would also be fine.

Reply:
hi @OliverSkroblin i will do this ASAP

Reply:
I created a UT to check to constructor and merge function from the changeset